### ISSUE TITLE
Add option for In-Person checkout

### ISSUE THREAD

**Issue by brewerc1** [grokability/snipe-it#7817]
We have a central asset checkout office, where employees visit to pick up hardware. 

Hardware must be signed for when checked out, by the person checking it out.

I'd love to see the checkout support this flow:

1. Asset is scanned by Admin User using an Asset Scanning iPad.
2. Admin User selects the Checkout User (or adds the new User), and chooses "In Person Checkout".
3. An interface screen is show for the Checkout User to enter a password.
4. A signature interface is displayed to the Checkout User. 
5. Admin turns over the requested hardware to the Checkout User.

The current checkout flow doesn't really support our process, because sending an email to a user requesting a signature confirmation *after* receiving the hardware is a fairly large hole in the accountability aspect.

**Comment by thesupporthero** [comment]
I plus one this. Our most popular method of people checking out equipment is in person. Our C-suite peeps really really want them to still sign for the item, and email them the agreement notice to them.
I had thoughts of creating an app that would do something similar that used SnipeIT api but...we had bigger fish to fry. ♥

**Comment by pbrink231** [comment]
We have this same issue.  And we’re thinking about doing it with an app or a page to select a user and then checkout assets, accessories, etc

**Comment by brewerc1** [comment]
Yes, this is a feature request.

**Comment by Relix360** [comment]
+1

**Comment by Relix360** [comment]
Yes

**Comment by brewerc1** [comment]
+1 for a feature

**Comment by t4ov** [comment]
+1

**Comment by loundsje** [comment]
Yes! This feature would be extremely helpful to our business processes.

**Comment by ArbiterGR** [comment]
We would love this as well. 

We have a similar workflow as the OP does and that would be most helpful. A way that this could be implemented, would be via the Unaccepted Assets menu, where a third action button could be added, to sign the item in person. 

This way we could have a tablet that the person accepting an asset could use to sign the asset acceptance. 

A user should accept the asset before taking it into his hands, not after.

**Comment by jellovator** [comment]
Please add this feature. This is our organization's preferred method (in-person hardware checkout, must sign form before asset is released to user).

**Comment by snipe** [comment]
Duplicate of #3750
